apps/site/src/app/about/TeamCard.tsx

```diff
@@ -19,8 +19,8 @@ import styles from "./TeamCard.module.scss";
 interface TeamCardProps {
 	name: string;
 	position: string;
-	image: string;
-	linkedInUrl: string;
+	image: string | undefined;
+	linkedInUrl: string | undefined;
```

Comment:
Chore: optional properties can be denoted with `?`.
Thought: `null` might make more sense semantically, but I'm not sure how the query output is parsed.